### PR TITLE
Add more meta-data to tokens response and add more doc responses

### PR DESCRIPTION
**Pull request by fragosti**
Docs https://github.com/0xProject/website/pull/218
### /swap/v0
```
 {"message":"This is the root of the Swap API. Visit https://0x.org/docs/api#swap for details about this API."}
```

### /swap/v0
```
{"message":"This is the root of the Standard Relayer API. Visit https://0x.org/docs/api#sra for details about this API."}
```

### /swap/v0/tokens
```
{
  "records": [
    {
      "symbol": "DAI",
      "address": "0x6b175474e89094c44da98b954eedeac495271d0f",
      "name": "Dai Stablecoin",
      "decimals": 18
    },
    {
      "symbol": "REP",
      "address": "0x1985365e9f78359a9B6AD760e32412f4a445E862",
      "name": "Augur Reputation",
      "decimals": 18
    },
    {
      "symbol": "WETH",
      "address": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
      "name": "Wrapped Ether",
      "decimals": 18
    },
    {
      "symbol": "ZRX",
      "address": "0xe41d2489571d322189246dafa5ebde1f4699f498",
      "name": "0x Protocol Token",
      "decimals": 18
    },
    {
      "symbol": "USDC",
      "address": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
      "name": "USD Coin",
      "decimals": 6
    },
    {
      "symbol": "BAT",
      "address": "0x0d8775f648430679a709e98d2b0cb6250d2887ef",
      "name": "Basic Attention Token",
      "decimals": 18
    },
    {
      "symbol": "MKR",
      "address": "0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2",
      "name": "Maker",
      "decimals": 18
    },
    {
      "symbol": "WBTC",
      "address": "0x2260fac5e5542a773aa44fbcfedf7c193bc2c599",
      "name": "Wrapped BTC",
      "decimals": 8
    },
    {
      "symbol": "SNX",
      "address": "0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f",
      "name": "Synthetix Network Token",
      "decimals": 18
    },
    {
      "symbol": "SUSD",
      "address": "0x57ab1e02fee23774580c119740129eac7081e9d3",
      "name": "sUSD",
      "decimals": 18
    },
    {
      "symbol": "KNC",
      "address": "0xdd974d5c2e2928dea5f71b9825b8b646686bd200",
      "name": "Kyber Network Crystal",
      "decimals": 18
    },
    {
      "symbol": "BNT",
      "address": "0x1f573d6fb3f13d689ff844b4ce37794d79a7ff1c",
      "name": "Bancor Network Token",
      "decimals": 18
    },
    {
      "symbol": "GNO",
      "address": "0x6810e776880c02933d47db1b9fc05908e5386b96",
      "name": "Gnosis Token",
      "decimals": 18
    },
    {
      "symbol": "LINK",
      "address": "0x514910771af9ca656af840dff83e8264ecf986ca",
      "name": "Chainlink Token",
      "decimals": 18
    },
    {
      "symbol": "REN",
      "address": "0x408e41876cccdc0f92210600ef50372656052a38",
      "name": "Republic Protocol",
      "decimals": 18
    },
    {
      "symbol": "GNT",
      "address": "0xa74476443119a942de498590fe1f2454d7d4ac0d",
      "name": "Golem Network Token",
      "decimals": 18
    },
    {
      "symbol": "OMG",
      "address": "0xd26114cd6ee289accf82350c8d8487fedb8a0c07",
      "name": "OmiseGO",
      "decimals": 18
    },
    {
      "symbol": "ANT",
      "address": "0x960b236a07cf122663c4303350609a66a7b288c0",
      "name": "Aragon Network Token",
      "decimals": 18
    },
    {
      "symbol": "SAI",
      "address": "0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359",
      "name": "Sai Stablecoin v1.0",
      "decimals": 18
    },
    {
      "symbol": "CVL",
      "address": "0x01fa555c97d7958fa6f771f3bbd5ccd508f81e22",
      "name": "Civil Token",
      "decimals": 18
    },
    {
      "symbol": "DTH",
      "address": "0x5adc961d6ac3f7062d2ea45fefb8d8167d44b190",
      "name": "Dether",
      "decimals": 18
    },
    {
      "symbol": "FOAM",
      "address": "0x4946fcea7c692606e8908002e55a582af44ac121",
      "name": "FOAM",
      "decimals": 18
    },
    {
      "symbol": "AST",
      "address": "0x27054b13b1b798b345b591a4d22e6562d47ea75a",
      "name": "AirSwap Token",
      "decimals": 4
    },
    {
      "symbol": "AION",
      "address": "0x4ceda7906a5ed2179785cd3a40a69ee8bc99c466",
      "name": "Aion Network",
      "decimals": 8
    },
    {
      "symbol": "GEN",
      "address": "0x543ff227f64aa17ea132bf9886cab5db55dcaddf",
      "name": "DAOstack",
      "decimals": 18
    },
    {
      "symbol": "STORJ",
      "address": "0xb64ef51c888972c908cfacf59b47c1afbc0ab8ac",
      "name": "Storj",
      "decimals": 8
    },
    {
      "symbol": "MANA",
      "address": "0x0f5d2fb29fb7d3cfee444a200298f468908cc942",
      "name": "Decentraland",
      "decimals": 18
    },
    {
      "symbol": "ENTRP",
      "address": "0x5bc7e5f0ab8b2e10d2d0a3f21739fce62459aef3",
      "name": "Hut34 Entropy Token",
      "decimals": 18
    },
    {
      "symbol": "MLN",
      "address": "0xbeb9ef514a379b997e0798fdcc901ee474b6d9a1",
      "name": "Melon",
      "decimals": 18
    },
    {
      "symbol": "LOOM",
      "address": "0xa4e8c3ec456107ea67d3075bf9e3df3a75823db0",
      "name": "Loom Network Token",
      "decimals": 18
    },
    {
      "symbol": "CELR",
      "address": "0x4f9254c83eb525f9fcf346490bbb3ed28a81c667",
      "name": "Celer Network Token",
      "decimals": 18
    },
    {
      "symbol": "RLC",
      "address": "0x607f4c5bb672230e8672085532f7e901544a7375",
      "name": "iExec RLC Token",
      "decimals": 9
    },
    {
      "symbol": "ICN",
      "address": "0x888666ca69e0f178ded6d75b5726cee99a87d698",
      "name": "ICONOMI",
      "decimals": 18
    },
    {
      "symbol": "DGD",
      "address": "0xe0b7927c4af23765cb51314a0e0521a9645f0e2a",
      "name": "Digix",
      "decimals": 9
    },
    {
      "symbol": "ZIL",
      "address": "0x05f4a42e251f2d52b8ed15e9fedaacfcef1fad27",
      "name": "Zilliqa",
      "decimals": 12
    },
    {
      "symbol": "cBAT",
      "address": "0x6c8c6b02e7b2be14d4fa6022dfd6d75921d90e4e",
      "name": "Compound Basic Attention Token",
      "decimals": 8
    },
    {
      "symbol": "cSAI",
      "address": "0x5d3a536e4d6dbd6114cc1ead35777bab948e3643",
      "name": "Compound Dai",
      "decimals": 8
    },
    {
      "symbol": "cSAI",
      "address": "0xf5dce57282a584d2746faf1593d3121fcac444dc",
      "name": "Compound Sai (Legacy Dai)",
      "decimals": 8
    },
    {
      "symbol": "cETH",
      "address": "0x4ddc2d193948926d02f9b1fe9e1daa0718270ed5",
      "name": "Compound Ether",
      "decimals": 8
    },
    {
      "symbol": "cREP",
      "address": "0x158079ee67fce2f58472a96584a73c7ab9ac95c1",
      "name": "Compound Augur",
      "decimals": 8
    },
    {
      "symbol": "cUSDC",
      "address": "0x39aa39c021dfbae8fac545936693ac917d5e7563",
      "name": "Compound USD Coin",
      "decimals": 8
    },
    {
      "symbol": "cZRX",
      "address": "0xb3319f5d18bc0d84dd1b4825dcde5d5f7266d407",
      "name": "Compound 0x",
      "decimals": 8
    },
    {
      "symbol": "0xBTC",
      "address": "0xb6ed7644c69416d67b522e20bc294a9a9b405b31",
      "name": "0xBitcoin Token",
      "decimals": 8
    },
    {
      "symbol": "SNT",
      "address": "0x744d70fdbe2ba4cf95131626614a1763df805b9e",
      "name": "Status Network Token",
      "decimals": 18
    },
    {
      "symbol": "SPANK",
      "address": "0x42d6622dece394b54999fbd73d108123806f6a18",
      "name": "SPANK",
      "decimals": 18
    },
    {
      "symbol": "BOOTY",
      "address": "0x6b01c3170ae1efebee1a3159172cb3f7a5ecf9e5",
      "name": "BOOTY",
      "decimals": 18
    },
    {
      "symbol": "BNB",
      "address": "0xb8c77482e45f1f44de1745f52c74426c631bdd52",
      "name": "Binance Coin",
      "decimals": 18
    },
    {
      "symbol": "ICX",
      "address": "0xb5a5f22694352c15b00323844ad545abb2b11028",
      "name": "ICON",
      "decimals": 18
    },
    {
      "symbol": "NMR",
      "address": "0x1776e1f26f98b1a5df9cd347953a26dd3cb46671",
      "name": "Numeraire",
      "decimals": 18
    },
    {
      "symbol": "GUSD",
      "address": "0x056fd409e1d7a124bd7017459dfea2f387b6d5cd",
      "name": "Gemini Dollar",
      "decimals": 2
    },
    {
      "symbol": "FUN",
      "address": "0x419d0d8bdd9af5e606ae2232ed285aff190e711b",
      "name": "FunFair",
      "decimals": 8
    },
    {
      "symbol": "PAX",
      "address": "0x8e870d67f660d95d5be530380d0ec0bd388289e1",
      "name": "PAX Stablecoin",
      "decimals": 18
    },
    {
      "symbol": "TUSD",
      "address": "0x0000000000085d4780b73119b644ae5ecd22b376",
      "name": "TrueUSD",
      "decimals": 18
    },
    {
      "symbol": "LPT",
      "address": "0x58b6a8a3302369daec383334672404ee733ab239",
      "name": "Livepeer",
      "decimals": 18
    },
    {
      "symbol": "ENJ",
      "address": "0xf629cbd94d3791c9250152bd8dfbdf380e2a3b9c",
      "name": "EnjinCoin",
      "decimals": 18
    },
    {
      "symbol": "POWR",
      "address": "0x595832f8fc6bf59c85c527fec3740a1b7a361269",
      "name": "PowerLedger",
      "decimals": 6
    },
    {
      "symbol": "REQ",
      "address": "0x8f8221afbb33998d8584a2b05749ba73c37a938a",
      "name": "Request",
      "decimals": 18
    },
    {
      "symbol": "DNT",
      "address": "0x0abdace70d3790235af448c88547603b945604ea",
      "name": "district0x",
      "decimals": 18
    },
    {
      "symbol": "MATIC",
      "address": "0x7d1afa7b718fb893db30a3abc0cfc608aacfebb0",
      "name": "Matic Network Token",
      "decimals": 18
    },
    {
      "symbol": "LRC",
      "address": "0xbbbbca6a901c926f240b89eacb641d8aec7aeafd",
      "name": "Loopring",
      "decimals": 18
    },
    {
      "symbol": "RDN",
      "address": "0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6",
      "name": "Raiden Network Token",
      "decimals": 18
    },
    {
      "symbol": "USDT",
      "address": "0xdac17f958d2ee523a2206206994597c13d831ec7",
      "name": "Tether USD",
      "decimals": 6
    }
  ]
}
```

### CODE REVIEW

File: src/handlers/sra_handlers.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { schemaUtils } from '../utils/schema_utils';
 export class SRAHandlers {
     private readonly _orderBook: OrderBookService;
     public static rootAsync(_req: express.Request, res: express.Response): void {
-        const message = `This is the root of the Standard Relayer API. Visit https://github.com/0xProject/standard-relayer-api/blob/master/http/v3.md for details about this API.`;
+        const message = `This is the root of the Standard Relayer API. Visit https://0x.org/docs/api#sra for details about this API.`;
```
Comment (xianny) [inline src/handlers/sra_handlers.ts]: Nit: put these doc URLs in the constants file

File: src/handlers/swap_handlers.ts [hunk 2]
```diff
@@ -17,6 +17,10 @@ import { findTokenAddress, isETHSymbol } from '../utils/token_metadata_utils';
 
 export class SwapHandlers {
     private readonly _swapService: SwapService;
+    public static rootAsync(_req: express.Request, res: express.Response): void {
+        const message = `This is the root of the Swap API. Visit https://0x.org/docs/api#swap for details about this API.`;
```
Comment (xianny) [inline src/handlers/swap_handlers.ts]: same here

File: src/routers/swap_router.ts [hunk 3]
```diff
@@ -8,6 +8,7 @@ import { SwapService } from '../services/swap_service';
 export function createSwapRouter(swapService: SwapService): express.Router {
     const router = express.Router();
     const handlers = new SwapHandlers(swapService);
+    router.get('/', asyncHandler(SwapHandlers.rootAsync.bind(SwapHandlers)));
```
Comment (xianny) [inline src/routers/swap_router.ts]: confused about what's going on here, should it be `asyncHandler(handlers.rootAsync.bind(handlers)));` ?

Comment (fragosti) [reply src/routers/swap_router.ts]: Nope.

**Pull request merged**